### PR TITLE
Fix/project export

### LINKED ISSUE #52: Project export can cause bad gateway error corrupting the download
# Issue
Saving a huge project will result in a corrupted file download if the application is deployed in an orchestration system (Kubernetes).
Furthermore, the export takes a lot of time and the end-user can be confused because the R session is not responsible anymore.

# Steps to reproduce
Visit metfamilies page, load a huge project (example files are provided) and try to export the project.

# Solution
The connection error is caused by a 502 error of the ingress component because the shiny-server is not responsible (the R session does not respond because of its high load). 
A better solution would be to preprocess the files that should be exported and then offer a download instead of opening the HTTP connection before.

### PR DESCRIPTION
**Pull request by culpinnis**
This PR closes #52

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -29,7 +30,7 @@ Suggests:
     testthat
 VignetteBuilder: knitr
 Maintainer: Hendrik Treutler<hendrik.treutler@ipb-halle.de>
-Description: DiffLogo is an easy-to-use tool for the discovery of regulated metabolite families.
+Description: We present a novel approach for the untargeted discovery of metabolite families offering a bird's eye view of metabolic regulation in comparative metabolomics. We implemented the presented methodology in the easy-to-use web application MetFamily to enable the analysis of comprehensive metabolomics studies for all researchers worldwide.
```
Comment (sneumann) [inline DESCRIPTION]: Well spotted !

File: MetFamily/ui.R [hunk 2]
```diff
@@ -1,1232 +1,1243 @@
...
-                      selectInput(multiple = FALSE, inputId = "metaboliteFamilyComparisonClass", label = "Compare to class", selected = "[init]", choices = c("[init]"), selectize = FALSE)
-                    ),
-                    plotOutput(height = 250, 
-                               outputId = "fragmentPlot2", 
-                               #hover    = "fragmentPlot2_hover",
-                               hover    = hoverOpts(
-                                 id = "fragmentPlot2_hover",
-                                 delay = 50, 
-                                 delayType = "debounce"
-
```
Comment (sneumann) [inline MetFamily/ui.R]: Is this a mix of reformatting and some actual change ? Can't review here.

Comment (culpinnis) [inline MetFamily/ui.R]: The real change of this PR

Comment (culpinnis) [reply MetFamily/ui.R]: https://github.com/ipb-halle/MetFamily/pull/54/files/3e41e23cdae91eb739e0309f9826253d7b7a99c0#r384733331

Whhopps. I have not noticed the change of the whole file.  Indeed only one line was changed.

**Pull request merged**